types/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "types-generated",
```

Comment:
Can we rename this to something meaningful? Like devlaunchers-strapi.

Reply:
done